lib/views/profile/settings/faqs_screen.dart

```diff
@@ -1,5 +1,5 @@
-import 'package:advance_pdf_viewer/advance_pdf_viewer.dart';
 import 'package:flutter/material.dart';
```

Comment:
You have done a lot of changes in this file, attach a video of this new view in the comments